server/lib/davIdIcon.js

```diff
@@ -1,5 +1,16 @@
 const getDavIdIconUrl = () => 'http://lorempixel.com/100/100/abstract/';
+const { URL } = require('url');
```

Comment:
There's no need to modify the original file for testing purposes.

Keep all the test related code in the test file.

Reply:
Ok, I have updated the files, will push changes now